### ISSUE TITLE
[Bug] Parentheses missing in DGMR grid loss function

### ISSUE THREAD

**Issue by nlstrotjo** [meteofrance/mfai#126]
In https://github.com/meteofrance/mfai/blob/main/mfai/pytorch/losses/gan_dgmr.py Line 45 is

        return difference / targets.size(1) * targets.size(3) * targets.size(4)

This corresponds to the error divided by the number of lead times, but it's then MULTIPLIED by the number of pixels. It should be divided, to get the error per pixel. So this is the corrected version:

        return difference / ( targets.size(1) * targets.size(3) * targets.size(4) )

The wrong loss function returns grid losses of ~1e13 and the GAN training breaks down, because the generator and discriminator losses do not matter anymore.
